Docs: instruct how to add your own volumes to Docker (#12189)
Automattic/jetpack: docker/README.md

- Server password: `wordpress`
- WordPress installation path: `/var/www/html`

## Custom plugins & themes in the container

Jetpack Docker environment can be wonderful for developing your own plugins and themes, too.

Since everything under `mu-plugins` and `wordpress/wp-content` are git-ignored, you'll want to keep those folders outside Jetpack repository folder and link them as volumes to your Docker instance.

1. First ensure your containers are stopped (`yarn docker:stop`).

> 1. Create a docker-compose file. You can place it anywhere in your computer:

Reviewer: Maybe it should be `2.` here?

Author: Sure, just handy to add all as `1.`s since in Markdown this gets rendered as "1. 2. 3." anyway; avoids annoyance of re-organising indexing when re-organising large lists.

This is just 3 items so I'll change it. :-)

Reviewer: That's a smart trick! Feel free to leave it as it is.

	```yml
	version: '3.3'
	services:
	  wordpress:
	    volumes:

> 	      - ~/my-plugin:/var/www/html/wp-content/plugins/my-plugin

Reviewer: Is `~/my-plugin` where my own plugin would be developed on my machine, and `/var/www/html/wp-content/plugins/my-plugin` where it would be linked?

Author: Yep! Do you recon I should clarify that more? I expected it to be clear from path names but I know our Docker setup inside-out so I'm blind to things like these.

Author: Elaborated more in https://github.com/Automattic/jetpack/pull/12189/commits/52b18dbcd2ffe69e07d67b82a308bb4e3f03aee1

Reviewer: Thank you!

	```
1. Start containers and include your custom volumes by running:
	```bash

> 	yarn docker:compose -f ~/docker-compose.my-volumes.yml up

Reviewer: In this case, it will start the Jetpack containers and my own additional one as well? I don't need to specify the Jetpack one there?

Author: Yep, this is same as `yarn docker:up` which spins up all the containers. We just want to use the underlying `yarn docker:compose` command instead because we need to pass `-f` attribute _before_ `up` command. Ngrok command does something similar.

Author: >and my own additional one as well

So to clarify; no additional containers. This will _expand_ `volumes` list in existing `wordpress` container because Docker-compose combines multiple configs into one.

- Server password: `wordpress`
- WordPress installation path: `/var/www/html`

## Custom plugins & themes in the container

Jetpack Docker environment can be wonderful for developing your own plugins and themes, too.

Since everything under `mu-plugins` and `wordpress/wp-content` is git-ignored, you'll want to keep those folders outside Jetpack repository folder and link them as volumes to your Docker instance.

1. First ensure your containers are stopped (`yarn docker:stop`).
2. Create a docker-compose file. You can place it anywhere in your computer:
	```yml
	version: '3.3'
	services:
	  wordpress:
	    volumes:
	      - ~/my-plugin:/var/www/html/wp-content/plugins/my-plugin
	```

> 	First part before `:` is path in your system to your own plugin and second part is path inside the Docker container. You can just replace "my-plugin" with what ever folder name you'd like. If you are developing a theme, switch `plugins/` to `themes/`.

Reviewer: ```suggestion
	What comes before `:` is the path to your own plugin or theme, in your system. What comes after `:` is the path inside the Docker container. You can replace `plugins/my-plugin` with the path to your own plugin or theme.
```

Author: Excellent, thanks! Feel free to commit further changes directly to PR :-)
